src/main/java/frc/robot/subsystems/PrimeSwerveModuleSubsystem.java

```diff
@@ -88,13 +97,11 @@ public void setDesiredState(SwerveModuleState desiredState){
       
    var currentVelocityInRotationsPer20ms = RobotMap.kDriveGearRatio * ((mDriveMotor.getSelectedSensorVelocity(0) / 5) / mEncoder.kPositionsPerRotation);
    var currentVelocityInMetersPer20ms = RobotMap.kDriveWheelCircumference * currentVelocityInRotationsPer20ms;
-   var desiredVelocity = (desiredState.speedMetersPerSecond / 50) * 2048;   var driveFeedForward = mDriveFeedforward.calculate(currentVelocityInMetersPer20ms, desiredVelocity, 0.2);
-   var driveFeedback = mDrivePIDController.calculate(currentVelocityInMetersPer20ms, desiredVelocity);
-   var desiredMotorVelocity = driveFeedback + driveFeedForward;
-   
-   mDriveMotor.set(ControlMode.PercentOutput, MathUtil.applyDeadband(desiredMotorVelocity, 0.15));
-   
-   
+   var desiredVelocity20ms = (desiredState.speedMetersPerSecond / 50) * 2048; 
```

Comment:
It looks like the literal integer `2048` is used several times in this block. Consider using a constant for this number